CASMINST-4271 streamline install improvements in docs for 1.2 (#1200)
Cray-HPE/docs-csm: install/bootstrap_livecd_remote_iso.md

   ```

1. <a name="set-up-site-link"></a>Set up the site-link, enabling SSH to work. You can reconnect with SSH after this step.

>    > **`Note:`** If your site's network authority or network administrator has already provisioned a DHCP given IPv4 address for your master node(s) external NIC(s), **then skip this step**.

Reviewer: I see all of the notes and warnings in this PR are both bolded and surrounded in backticks. In all of the /operations documentation (and in other product stream documentation), these are only bolded. 

I think we should keep all of these as bolded only to be consistent.

Author: Thats an interesting and valid observation. These install docs however had the Note and Warnings using this existing style already. At this time we're trying to limit from making too many style, format and content changes at once that might confuse readers. For now, we're going to move ahead with this style. We are adding a linter to the docs and will review this proposed change to remove the back tics and leave just the bold at a later time.

>    > **`Note:`** If your site's network authority or network administrator has already provisioned a DHCP given IPv4 address for your master node(s) external NIC(s), **then skip this step**.

Reviewer: I see all of the notes and warnings in this PR are both bolded and surrounded in backticks. In all of the /operations documentation (and in other product stream documentation), these are only bolded. 

I think we should keep all of these as bolded only to be consistent.

Author: Thats an interesting and valid observation. These install docs however had the Note and Warnings using this existing style already. At this time we're trying to limit from making too many style, format and content changes at once that might confuse readers. For now, we're going to move ahead with this style. We are adding a linter to the docs and will review this proposed change to remove the back tics and leave just the bold at a later time.

   1. Generate the system configuration

>       > **`Note:`** for those more familiar with a CSM Install, this step may be skipped entirely by simple invoking pit-intas detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.

Reviewer: ```suggestion
      > **`Note:`** For those more familiar with a CSM install, this step may be skipped entirely by simple invoking `pit-intas` detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
```

>       > **`Note:`** for those more familiar with a CSM Install, this step may be skipped entirely by simple invoking pit-intas detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.

Reviewer: ```suggestion
      > **`Note:`** For those more familiar with a CSM install, this step may be skipped entirely by simple invoking `pit-intas` detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
```
```suggestion
      > **`Note:`** for those more familiar with a CSM Install, this step may be skipped entirely by simple invoking pit-intas detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
```

   1. Check for the configuration files. The needed files should be in the current directory.

      > **`Note:`** for those more familiar with a CSM Install, this step may be skipped entirely by simple invoking pit-intas detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.

      ```bash
      pit:/var/www/ephemeral/prep/# ls -1
      ```

>        1. Expected output looks similar to the following:

Reviewer: This shouldn't be a separate sub-step as it is just an example of output and the user is not doing an action. It should just be indented under the step above.

>        1. Expected output looks similar to the following:

Reviewer: This shouldn't be a separate sub-step as it is just an example of output and the user is not doing an action. It should just be indented under the step above.

      ```

   1. Generate the system config:

>       > **`Note:`** the provided command below is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significatnly depending on ones system and site configuration.

Reviewer: ```suggestion
      > **`Note:`** The following command is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significantly depending on the system and site configuration in use.
```

>       > **`Note:`** the provided command below is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significatnly depending on ones system and site configuration.

Reviewer: ```suggestion
      > **`Note:`** The following command is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significantly depending on the system and site configuration in use.
```
```suggestion
      > **`Note:`** the provided command below is an **example only**, run `csi config init --help` to print a full list of parameters that must be set. These will vary significatnly depending on ones system and site configuration.
```

<a name="prepare-site-init"></a>
#### 4.2 Prepare Site Init

1. First, prepare a shim to facilitate going through the site-init guide:

    ```bash
    pit# mkdir -vp /mnt/pitdata
    pit# mount -v --bind /var/www/ephemeral/ /mnt/pitdata/
    ```

1. Follow the procedures to [Prepare Site Init](prepare_site_init.md) directory for your system.

> 1. Finally, cleanup the shim:

Reviewer: ```suggestion
1. Clean-up the shim:
```

> 1. Finally, cleanup the shim:

Reviewer: ```suggestion
1. Clean-up the shim:
```
